tide_site.install

```diff
@@ -347,3 +348,44 @@ function tide_site_update_8018() {
     $field->setSetting('file_extensions', 'png gif jpg jpeg svg')->save();
   }
 }
+
+/**
+ * Changes internal:/node/{id} to entity:node/{id}.
```

Comment:
@vincent-gao What's the effect of making this change?

Reply:
Hi @GROwen ,
I made https://github.com/dpc-sdp/tide_site/pull/72/files#diff-6e9b0ed520e9f281aea3d4a69a24137fR720 , so I thought i should add an update hook accordance with the `presave` change.

UI effect:
- Before
![image](https://user-images.githubusercontent.com/8788145/93147793-8a123d80-f735-11ea-855d-a9aa5419326b.png)
- After
![image](https://user-images.githubusercontent.com/8788145/93147812-98605980-f735-11ea-893d-c1785497f23b.png)

Search api effect:
@kurtfoster told me that it might take advantage of this change to get better performance of search api?(cannot clearly remember his statement.). it relates to search.